crates/block-producer/src/mempool/mod.rs

```diff
@@ -388,7 +398,7 @@ impl Mempool {
     fn revert_transactions(
         &mut self,
         txs: Vec<TransactionId>,
-    ) -> Result<(), GraphError<TransactionId>> {
+    ) -> Result<BTreeSet<TransactionId>, GraphError<TransactionId>> {
```

Comment:
Could we update doc comments to describe what this function returns?

Reply:
Updated the docs!